datasets.py

```diff
@@ -1,921 +0,0 @@
-import argparse
```

Comment:
Is this deleting the entire datasets.py file from the repo, or is it an artifact of weird merging?